### PR TITLE
New unit: Swiss Francs per kilogram

### PR DESCRIPTION
**Pull request by tpluscode**
I would like to propose a new unit which will be used at [Lindas Linked Data Services](https://lindas.admin.ch/) to publish Swiss open data

### CODE REVIEW

File: vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl [hunk 1]
```diff
@@ -8370,7 +8370,7 @@ unit:CHF-PER-KiloGM
   qudt:conversionMultiplierSN 1.0E-3 ;
```
Comment (steveraysteveray) [inline vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:8370]: Since the kilogram is the base SI unit, the conversion multipliers should be 1.0

**Review by steveraysteveray — CHANGES_REQUESTED** [review]
One item that is still missing is the quantity kind CostPerMass that your unit can then point to.

**Comment by tpluscode** [comment]
> One item that is still missing is the quantity kind CostPerMass that your unit can then point to.

Shall I add it in the PR?

File: vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl [hunk 2]
```diff
@@ -8367,7 +8367,8 @@ unit:CHF-PER-KiloGM
   a qudt:Unit ;
   qudt:applicableSystem sou:SI ;
   qudt:conversionMultiplier 0.001 ;
```
Comment (steveraysteveray) [inline vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:8369]: This one needs to be 1.0
The SN one needs to be 1.0E0

**Review by steveraysteveray — CHANGES_REQUESTED** [review]
The quantity kind and unit need triples for qudt:hasDimensionVector
Namely:
  qudt:hasDimensionVector qkdv:A0E0L0I0M-1H0T0D0 ;

Also, a brief triple for qudt:plainTextDescription.

See https://github.com/qudt/qudt-public-repo/wiki/QuantityKind-Vocabulary-Submission-Guidelines

**Comment by tpluscode** [comment]
Thank you for your pointers @steveraysteveray

**Review by steveraysteveray — APPROVED** [review]

**Pull request merged**